Example/src/screens/SearchBar.tsx

```diff
@@ -46,6 +46,7 @@ const MainScreen = ({ navigation }: MainScreenProps): JSX.Element => {
   const [autoCapitalize, setAutoCapitalize] =
     useState<AutoCapitalize>('sentences');
   const [inputType, setInputType] = useState<InputType>('text');
+  const [shouldClearOnSubmit, setShouldClearOnSubmit] = useState(false);
```

Comment:
I think `clearOnSubmit` would be a better name for the state and property overall.